src/en/models-config.md

```diff
@@ -70,7 +74,7 @@ ca-cert | string |  |  | The certificate of the CA that signed the state server
 ca-cert-path | string |  |  | Path to file containing CA certificate
 ca-private-key | string |  |  | The private key of the CA that signed the state server certificate, in PEM format
 ca-private-key-path | string |  |  | Path to file containing CA private key
-default-series | string |  | valid series name, e.g. 'trusty' | The default series of Ubuntu to use for deploying charms
+default-series | string |  | valid series name, e.g. 'xenial' | The default series of Ubuntu to use for deploying charms
 development | bool | false |  | Whether the model is in development mode
```

Comment:
I wonder when the table format got fucked up. I had all this spaced out so it was easy to add/amend things without getting lost :(